python/tk_multi_launchapp/base_launcher.py

```diff
@@ -287,6 +292,33 @@ def _launch_app(
             del sys.path[:]
             sys.path.extend(sys_path_clone)
 
+    def launch_indicator(self, app_path):
+        """
+        This displays a temporary frameless QDialog with an overlay
+        widget embedded indicating that the command used to
+        start the DCC was successfully executed.
+
+        :param app_path: Full path name to the DCC. This may contain environment
+                 variables and/or the locally supported {version}, {v0},
+                 {v1}, ... variables
+        """
+        if self._tk_app.engine.has_ui:
+            from ..launch_indicator_dialog import populate_launch_dialog
+
+            wid, dial = populate_launch_dialog(self._tk_app)
+            # Start spinner
+            wid.start_progress()
+            # Report progress
+            splash_message = "Launching executable '%s'" % (app_path)
+            wid.report_progress(0.00, splash_message)
+            splash_msg = "Launched successfully"
+            QtCore.QTimer.singleShot(
+                7000,
+                lambda: wid.report_progress(0.97, splash_msg),
+            )
+            # Hide the QDialog
+            QtCore.QTimer.singleShot(10000, lambda: dial.hide())
```

Comment:
I'm not sure how QT works here, but is the dialog being destroyed (freed memory) or just being hidden? Do you know if this has an impact on performance?

Reply:
Good catch, thanks for the suggestion !, I have changed it by this method reject() which looks like triggers a close event and delete the dialog while it is visible.